app/lib/accounts.js

```diff
@@ -1,7 +1,7 @@
 const userEmailSchema = require("./userEmailSchema");
 AccountsTemplates.configure({
   // Behavior
-  forbidClientAccountCreation: true,
+  forbidClientAccountCreation: false,
```

Comment:
We don't yet want to change this setting permanently.

Reply:
Noted.

Reply:
OK, go ahead and revert this change.